### PR TITLE
updated start_colab.md

### PR DESCRIPTION
**Pull request by hotessy**
added instructions for using GDrive as a permanent data store for Colab.

### CODE REVIEW

**Comment by hotessy** [comment]
Added some instructions so that people using Google Colab can permanently store data files using Google Drive.
Please let me know if the changes are suitable and if they need to updated in some other file as well.

File: docs/start_colab.md [hunk 1]
```diff
@@ -43,14 +43,31 @@ If you are returning to work and have previously completed the steps below, plea
 
     <img alt="stop" src="/images/colab/08.png" height="480" class="screenshot">
 
-### Step 3: Saving your work
+### Step 3: Saving your notebook
```
Comment (hotessy) [inline docs/start_colab.md]: Changed `work` to `notebook` so that there is no confusion in terminology mentioned in step 4

**Comment by jph00** [comment]
Very helpful!

**Comment by hotessy** [comment]
Awesome!

**Pull request merged**